src/fields/hidden-field.ts

```diff
@@ -0,0 +1,9 @@
+import * as Yup from "yup";
+import { IFieldSchemaBase } from "../schema-generator";
+import { IFieldGenerator } from "./types";
+
+export interface IHiddenFieldSchema<V = undefined> extends IFieldSchemaBase<"hidden-field", V> {}
+
+export const hiddenField: IFieldGenerator<IHiddenFieldSchema> = (id, { validation }) => ({
+	[id]: { yupSchema: Yup.string(), validation },
```

Comment:
actually on FEE `valueType` can be configured to support number and boolean as well, so the base yup schema should be based on this prop